contracts/MoneyBox.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Can we push all the MoneyBox contracts into the mixin folder? I think they are just "utility for us" and don't belong to the core.

Reply:
fixed in [0556a3a](https://github.com/RealityETH/subjectivocracy/pull/98/commits/0556a3a1d09ebd3af8195dff81bc5e77fbd6f9cf)